### PR TITLE
TE-1463 / Adding all read-only admin test for NFS shares

### PR DESCRIPTION
**Pull request by ericbsd**
Cleaned and some code
removed some unused items in python_functions of pytest.ini

### CODE REVIEW

**Comment by ericbsd** [comment]
image

File: keywords/webui/nfs.py [hunk 1]
```diff
@@ -130,7 +216,19 @@ def click_add_networks_button(cls):
         COM.click_button('add-item-networks')
 
     @classmethod
-    def click_remove_from_list_button(cls):
+    def click_nfs_share_edit_button(cls, share_xpath: str) -> None:
```
Comment (markbainix) [inline keywords/webui/nfs.py:219]: FYI... in common_share keyword there is `click_edit_share()`

File: test_cases/webui/permissions/read-only-admin/test_read_only_admin_share.py [hunk 2]
```diff
@@ -45,27 +52,23 @@ def tear_down_test(self, data):
...
         # Verify the read-only admin can view pre-configured iSCSI shares
         assert Common_Shares.assert_share_card_displays('iscsi') is True
         assert Common_Shares.assert_iscsi_target_is_visible(data['iscsi_name_xpath']) is True
 
-    @allure.tag("Update", 'iSCSI')
-    @allure.story("Read Only Admin Is Not Able To Enable Or Disable The iSCSI Service")
-    def test_read_only_is_not_able_enable_or_disable_iscsi_service(self, data):
+    @allure.tag('Update', 'iSCSI')
+    @allure.story("Read Only Admin Is Not Able To Enable Or Disable The iSCSI Service On The iSCSI Card")
+    def test_read_only_is_not_able_enable_or_disable_iscsi_service_on_the_iscsi_card(self, data):
```
Comment (markbainix) [inline test_cases/webui/permissions/read-only-admin/test_read_only_admin_share.py:68]: able_enable -> able_to_enable

File: test_cases/webui/permissions/read-only-admin/test_read_only_admin_share.py [hunk 3]
```diff
@@ -18,16 +18,23 @@
 @pytest.mark.parametrize('data', get_data_list('read_only_admin_shares'), scope='class')
 class Test_Read_Only_Admin_Share:
```
Comment (markbainix) [inline test_cases/webui/permissions/read-only-admin/test_read_only_admin_share.py:19]: Suggest:
maybe split into separate files for 'iSCSI' and 'NFS' and 'SMB'  future runs may only need to execute against one type of share. reduce the need for running all tests.

**Review by markbainix — APPROVED** [review]

**Pull request merged**